scijava/scijava-struct/src/main/java/org/scijava/struct/Structs.java

```diff
@@ -69,26 +69,19 @@ public static void checkModifiers(String message, final ArrayList<ValidityProble
 		}
 	}
 
-	public static boolean checkValidity(Member<?> m, String name, Class<?> type,
-		boolean isFinal, Set<String> names, ArrayList<ValidityProblem> problems)
+	public static boolean checkValidity(Member<?> m, Supplier<String> name, Class<?> type,
```

Comment:
It feels odd to me that some arguments became `Supplier`, but not all arguments. But I'm going to punt on this for the moment, knowing that I'll be scrutinizing the `org.scijava.struct` API again prior to its release.

Reply:
Fixed in 325a71eb. I just reverted that change. We were only calling this method in one place, and it was just creating the `Supplier` in the call 😅